### PR TITLE
Add define-symbols recipe

### PR DESCRIPTION
**Pull request by wasamasa**
As requested on `#chicken`. I've taken the liberty of adding a new headline.

### CODE REVIEW

File: recipes/define-symbols.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+which evaluate to `x`, `y` and `z` respectively.
+
+## Solution
+
+```scheme
+(define-syntax make-symbols
+  (syntax-rules ()
+    ((_ symbol ...)
+     (begin
+       (define symbol 'symbol) ...))))
```
Comment (lassik) [inline recipes/define-symbols.md:16]: Missing the recursive call to `make-symbols`?

File: www-index.scm [hunk 2]
```diff
@@ -45,5 +45,8 @@
  ("Reflection"
   "get-type-of-object")
 
+ ("Syntactic extensions"
```
Comment (lassik) [inline www-index.scm:48]: How about just `Macros`?

**Comment by lassik** [comment]
Thanks for submitting this!

I assume the version using explicit-renaming macros is overkill?

File: recipes/define-symbols.md (hunk 1 above)
Comment (wasamasa) [reply recipes/define-symbols.md:16]: Why would a recursive call be necessary?

File: www-index.scm (hunk 2 above)
Comment (wasamasa) [reply www-index.scm]: Fixed

File: recipes/define-symbols.md (hunk 1 above)
Comment (lassik) [reply recipes/define-symbols.md:16]: Wow, I didn't know about the `syntax-rules` feature you're using.

This one?

> Pattern variables that occur in subpatterns followed by one or more instances of the identifier ellipsis are allowed only in subtemplates that are followed by as many instances of ellipsis. They are replaced in the output by all of the elements they match in the input, distributed as indicated.

**Comment by wasamasa** [comment]
Going by the repository README a R6RS solution with syntax-case would be preferable, so I chose to omit it.

**Comment by lassik** [comment]
Fair enough, though we're not strict about that.

**Pull request merged**